lib/MusicBrainz/Server/Controller/WS/2/Event.pm

```diff
@@ -95,10 +109,20 @@ sub event_browse : Private {
     } elsif ($resource eq 'collection') {
         $events = $self->browse_by_collection($c, 'event', $id, $limit, $offset);
     } elsif ($resource eq 'event') {
-        my $event = $c->model('Event')->get_by_gid($id);
-        $c->detach('not_found') unless $event;
+        if (ref($id) ne 'ARRAY') {
+            $id = [$id];
+        }
+        if (@$id > $MAX_EVENT_BROWSE_PARAMS) {
+            @$id = @$id[0 .. ($MAX_EVENT_BROWSE_PARAMS - 1)];
+        }
```

Comment:
First: Thank you for implementing.
Would it be better to fail here to make the user aware that they don't get everything they expect?

Reply:
That does sound preferable to me too, I think.

Reply:
Fixed.